news/2018-02-20-community-choice-2017-results.md

```diff
@@ -0,0 +1,83 @@
+
+The votes presented below are thus purely unweighted, raw voting totals for each particular map. One vote is simply one vote - one person's approvement and endorsement of that map as their most favourite of 2017.
+
+Without further ado, here are the four grand winners from each mode:
+
+## [osu!](https://osu.ppy.sh/community/contests/59)
+
+[![](/wiki/shared/news/2018-02-20-community-choice-2017-results/hitorigoto.jpg)](https://osu.ppy.sh/beatmapsets/596704)
+
+In an overwhelming display of support that should surprise exactly nobody, [Doormat](https://osu.ppy.sh/u/3230571)'s set of [ClariS - Hitorigoto -TV MIX-](https://osu.ppy.sh/beatmapsets/596704) takes the title for Community Favourite of 2017 with an outstanding **4434** votes, nearly a full thousand more than its nearest competitor.
+
+Riding off the (questionable) success of last year's hit anime Eromanga-sensei, Hitorigoto is a track all about talking to one's self, and this highly popular set certainly managed to get people talking.
+
+Notable runners up were:
+
+[Agnete Kjolsrud - Get Jinxed](http://osu.ppy.sh/beatmapsets/361740) by [Tarrasky](https://osu.ppy.sh/u/4098393) in 2nd place overall with **3565** votes.
+
+[\*namirin - Koi no Hime Hime Pettanko](http://osu.ppy.sh/beatmapsets/485056) by [C u r i](https://osu.ppy.sh/u/4746949) in 3rd place overall with **2749** votes.
+
+## [osu!taiko](https://osu.ppy.sh/community/contests/60)
+
+[![](/wiki/shared/news/2018-02-20-community-choice-2017-results/renren.jpg)](https://osu.ppy.sh/beatmapsets/663337)
+
+By the skin of its teeth with only **9** votes between first and second place, [Backfire](https://osu.ppy.sh/u/263110)'s set of [Sara - Ren Ren Ai Ai Cir Cir Cula Cula Tion Tion](https://osu.ppy.sh/beatmapsets/663337) snags the title of Community Favourite 2017 for osu!taiko with **117** votes.
+
+Unlike the track itself, we're definitely not stuttering.
+
+Notable runners up were:
+
+[D(ABE3) - MANIERA](http://osu.ppy.sh/beatmapsets/400761) by [\_yu68](https://osu.ppy.sh/u/6170507) with **108** votes.
+
+[Dunderpatrullen - We Are Number One (Remix)](https://osu.ppy.sh/beatmapsets/547301) by [Midnaait](https://osu.ppy.sh/u/1506011) with **104** votes.
+
+## [osu!catch](https://osu.ppy.sh/community/contests/61)
+
+[![](/wiki/shared/news/2018-02-20-community-choice-2017-results/onthefm.jpg)](https://osu.ppy.sh/beatmapsets/609679)
+
+Fiercely competitive all the way to the end, osu!catch had some of the most robust voting of all the gamemodes so far, a few big maps fighting for the lion's share of the vote.
+
+In the end, [Ascendance](https://osu.ppy.sh/u/2931883)'s set of [C\-Show - On the FM](https://osu.ppy.sh/beatmapsets/609679) triumphed over the others with **249** votes, and walks away with the title of Community Favourite 2017 for osu!!catch.
```

Comment:
why is the C-Show escaped or is it like that 🤔

Reply:
escaped due to -, just making sure

Reply:
i don't think it does anything if it's not on the line beginning, probably not?